command/release_list.go

```diff
@@ -72,6 +72,7 @@ Options:
                        - github (default)
                        - gitlab
                        - tfregistryModule
+                       - tfregistryProvider
```

Comment:
Could you add the same format example for the source argument as the release latest command?

Reply:
Done.